### ISSUE TITLE
Doesn't work on arduino mega 2560 (clone)

### ISSUE THREAD

**Issue by belpyro** [MCUdude/MegaCore#211]
I tried several megas, and none worked after the bootloader replacement. MiniCore works properly with UNO (cloned).
The error looks like:
```
Programmer Type : Urclock
         Description     : Urclock programmer for urboot bootloaders using urprotocol
         Urboot protocol for ATmega2560
avrdude: AVR device initialized and ready to accept instructions
avrdude ur_initstruct() error: unknown bootloader ... please specify -xbootsize=<num>

avrdude avr_signature() error: unable to read signature data for part ATmega2560, rc=-1
avrdude main() error: unable to read signature data, rc=-1
```
My settings are:
<img width="151" alt="image" src="https://github.com/MCUdude/MegaCore/assets/4113641/e9b4dea6-537a-470a-b408-666a3a6aacb7">

Could someone route me on the right way? Thx!
P.s. I need clarification that the bootloader is always the same. Fuses changed only. Is it correct?

**Comment by MCUdude** [comment]
... And you have burned the bootloader bundled with Megacore, using your USBasp programmer?

**Comment by belpyro** [comment]
> ... And you have burned the bootloader bundled with Megacore, using your USBasp programmer?

Yes. That's correct. As I mentioned before, the same approach for UNO works fine. Thx!

**Comment by MCUdude** [comment]
Did you ever figure this out? It seems like the bootloader installed is an older type not known to Avrdude, and that's why it won't work with the new `urclock` upload option. Did you ever try to re-flash thebootloader from the latest version of MegaCore?
